documentation/tutorials/check.md

```diff
@@ -99,6 +99,39 @@ We can then run `check` with the `-drift` flag:
 
 This creates another pair of HTML and JSON reports. This time the files contain a drift report, showing how the target database schema (in this case `foobar`) differs from whats contained in the applied migrations.
 
+### Programmatically identifying drift
+<h4> <i class="fa fa-windows"></i> Windows </h4>
+
+Powershell script to parse the output JSON,find the last drift report it contains and report the status of the flag
```

Comment:
Space after `JSON,` and maybe a colon at the end of the line?

Also could maybe improve the wording:
```
The following Powershell script parses...:
```